code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -3080,6 +3080,92 @@
 	required_reagents = list(FORMIC_ACID = 1, PHENOL = 1, RADIUM = 1)
 	result_amount = 3
 
+//Karmotrine Drinks
+
+/datum/chemical_reaction/smokyroom
+	name = "Smoky Room"
+	id = SMOKYROOM
+	result = SMOKYROOM
+	required_reagents = list(MANHATTAN = 1, ICE = 1, KARMOTRINE = 2)
+	result_amount = 4
+
+/datum/chemical_reaction/ragstoriches
+	name = "Rags to Riches"
+	id = RAGSTORICHES
+	result = RAGSTORICHES
+	required_reagents = list(VODKA = 1, COGNAC = 1, KARMOTRINE = 3)
+	result_amount = 5
+
+/datum/chemical_reaction/badtouch
+	name = "Bad Touch"
+	id = BAD_TOUCH
+	result = BAD_TOUCH
+	required_reagents = list(RUM = 2, KAHLUA = 1, ICE = 1, KARMOTRINE = 1)
+	result_amount = 5
+
+/datum/chemical_reaction/electricsheep
+	name = "Electric Sheep"
+	id = ELECTRIC_SHEEP
+	result = ELECTRIC_SHEEP
+	required_reagents = list(WINE = 1, SILICON = 1, KARMOTRINE = 3)
+	result_amount = 5
+
+/datum/chemical_reaction/suicide
+	name = "Suicide"
+	id = SUICIDE
+	result = SUICIDE
+	required_reagents = list(SPACEMOUNTAINWIND = 1, DR_GIBB = 1, COLA = 1, LEMON_LIME = 1, KARMOTRINE = 1)
+	result_amount = 5
+
+/datum/chemical_reaction/scientistsserendipity
+	name = "Scientist's Serendipity"
+	id = SCIENTISTS_SERENDIPITY
+	result = SCIENTISTS_SERENDIPITY
+	required_reagents = list(MOONROCKS = 1, KARMOTRINE = 1)
+	result_amount = 2
+
+/datum/chemical_reaction/metabuddy
+	name = "Metabuddy"
+	id = METABUDDY
+	result = METABUDDY
+	required_reagents = list(TONIC = 2, KARMOTRINE = 3)
+	result_amount = 5
+
+/datum/chemical_reaction/waifu
+	name = "Waifu"
+	id = WAIFU
+	result = WAIFU
+	required_reagents = list(SAKE = 1, KARMOTRINE = 4)
+	result_amount = 5
+
+/datum/chemical_reaction/beepskyclassic
+	name = "Beepsky Classic"
+	id = BEEPSKY_CLASSIC
+	result = BEEPSKY_CLASSIC
+	required_reagents = list(BEEPSKY_SMASH = 2, KARMOTRINE = 3)
+	result_amount = 5
+
+/datum/chemical_reaction/spider
+	name = "Spiders"
+	id = SPIDERS
+	result = SPIDERS
+	required_reagents = list(CLONEXADONE = 3, KARMOTRINE = 2)
+	result_amount = 5
+
+/datum/chemical_reaction/weedeater
+	name = "Weed Eater"
+	id = WEED_EATER
+	result = WEED_EATER
+	required_reagents = list(SPACE_DRUGS = 1, AMATOXIN = 1, PSILOCYBIN = 1, KARMOTRINE = 2)
+	result_amount = 5
+
+/datum/chemical_reaction/lemonlime
+	name = "Lemon Lime"
+	id = LEMON_LIME
+	result = LEMON_LIME
+	required_reagents = list(LIMEJUICE = 1, LEMONJUICE = 1, SODAWATER = 1)
+  result_amount = 3
```

Comment:
Shouldn't you press tab here?

Reply:
yeah, the github online editor fuckered me on that, whoops.